src/org/usfirst/frc/team2706/robot/commands/IntakeCube.java

```diff
@@ -53,7 +53,7 @@ public IntakeCube(Supplier<Double> speed, boolean sameRatio) {
         this.sameRatio = sameRatio;
 
         // FIXME: Sometimes stops auto modes from running correctly
-//        this.requires(Robot.intake);
```

Comment:
There's some other code that is commented out with the same style with the `//` at the start of the line. Should I change that too?

Reply:
I don't think it's a big deal. I changed it in a couple of places. If it's easy, then sure.